package.json

```diff
@@ -33,7 +33,8 @@
     "http": "static spec/jasmine -H '{\"Cache-Control\": \"no-cache, must-revalidate\"}'",
     "****** NODE *****": "",
     "postinstall": "cd docs && yarn",
-    "precommit": "lint-staged & grunt exec:docs_dist & wait && git add -A docs/{index.html,dist/**/*}"
+    "precommit": "lint-staged & grunt exec:docs_dist",
```

Comment:
@sampi Don't you think that perhaps the `wait` failed because it said `& wait` instead of `&& wait` ? I don't know shell scripting well enough to draw this conclusion myself.